docker/composer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer/composer
```

Comment:
Yes, composer is a dev dependency, which is also the reason for not including it in the harvester-container. Harvester works fine without having composer installed.

Composer checks for specific extensions in PHP, so it makes sense to tailer the composer-container to the project, allowing it to fail if an extension does not exist.

I, personally, like having everything i need for beginning work on a project as part of the docker-compose setup.

`composer/composer:alpine` is fine by me